### PR TITLE
Add a picker provider

### PR DESCRIPTION
**Pull request by dmolineus**
This pull requests adds the picker support for multilingual data containers.

### CODE REVIEW

**Comment by aschempp** [comment]
Can you explain what the picker class offers that the universal picker does not?

**Comment by dmolineus** [comment]
> I guess we need a custom picker because just to support our driver class, right?

Yes, the generic picker is limited to `DC_Table`, see https://github.com/contao/contao/blob/50502c09eec66d75b1c1ae280ceec90a0ab4ca62/core-bundle/src/Picker/TablePickerProvider.php#L26

**Comment by aschempp** [comment]
just let me know when you tested this and if I should release 4.3 right-away or you maybe have some more stuff coming up 🙃

**Comment by benfolds** [comment]
Is there a schedule for this pull request? Can i buy some coffee?

**Comment by dmolineus** [comment]
@aschempp I had to adjust the implementation to support both driver names `Multilingual` and the FCQN `Driver::class` in [7e8f3d1](https://github.com/terminal42/contao-DC_Multilingual/pull/74/commits/7e8f3d1e3c96be5c1b2f2d7453435a0ba84a63e6). I did not stumple upon any other issue so it's ready to merge from my point of view.

**Review by aschempp — CHANGES_REQUESTED** [review]

File: src/Driver.php [hunk 1]
```diff
@@ -131,7 +131,7 @@ public function edit($intId=null, $ajaxId=null)
             throw new InternalServerErrorException('Table "' . $this->strTable . '" is not editable.');
         }
 
-        if ($intId != '')
+        if ('' !== $intId && $intId !== null)
```
[the remark this answers is not in the document]
Comment (aschempp) [reply src/Driver.php]: changed in d5739f33e29c549f04b5af31e761022cedf411de to what the Contao Core does
https://github.com/contao/contao/blob/5.x/core-bundle/src/Resources/contao/drivers/DC_Table.php#L1783

File: src/Picker/MultilingualPickerProvider.php [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        $table = $this->getTableFromContext($context);
+
+        $this->framework->initialize();
+        $this->framework->createInstance(DcaLoader::class, [$table])->load();
+
+        if (0 === \count($this->getModulesForTable($table))) {
+            return false;
+        }
+
+        $driverClass = DataContainer::getDriverForTable($table);
```
[the remark this answers is not in the document]
Comment (aschempp) [reply src/Picker/MultilingualPickerProvider.php]: I have simplified the fix in 166879a1b3686e5581cff89fd92eafc6525f985b

**Review by aschempp — APPROVED** [review]

**Comment by aschempp** [comment]
Thank you @dmolineus

**Pull request merged**